sacraments.md

```diff
@@ -8,4 +8,40 @@ permalink: /sacraments
 
 All Sacraments are available to parishioners in English.
 
-(More information coming soon.)
+## Baptism
+Baptism for infants/children usually takes place during Sunday mass.
+For infant/child baptism, please contact the secretary of Sankt Elisabeth, Frau Stephania Trost at 040 696 3838 70 or [info@st-elisabeth-hh.de](mailto:info@st-elisabeth-hh.de).
+Please make contact at least 6 weeks before you would like the baptism to take place.
+After you have contacted Frau Trost and signed the registration form, Fr. Ritchille Salinas will contact you to discuss and plan the baptism.
+
+Baptism for adults is possible through the Rite of Christian Initiation of Adults (RCIA).
+If you are interested in joining the church as an adult, please contact Fr. Christian Modemann, SJ at [Catholic Faith Information](https://pfarrei-edith-stein.de/KGI/Team__3550) (KGI: Katholische Glaubensinformation) Hamburg at 0162 13 53 261 or [modemann@kgi-hh.de](mailto:modemann@kgi-hh.de).
+
+## Confession
+Confession is available every Sunday before mass starting at 11:00.
+Due to COVID-19, please pre-register with Fr. Ritchille Salinas at [svdsalinas@gmail.com](mailto:svdsalinas@gmail.com).
+Confessions are being heard in the parish hall (Gemeindehaus) during the pandemic.
+
+## Eucharist
+Children or adults who are interested in being prepared for first holy communion should contact Mario Stallone at [TBD](mailto:TBD) to find out about when the next round of classes will take place.
```

Comment:
A reminder to update this before merging this in

Reply:
Done. Thanks!